.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,11 @@
 -
 -
 
+## Reviewer's checklist / merge requirements:
```

Comment:
What about adding a check for the head branch not being `main`? (possibly with a less awkward phrasing than my suggestion below):

```suggestion
## Reviewer's checklist / merge requirements:
- [ ] The head branch (i.e. the "source" of the changes) is not the `main` branch on the PR author's fork
```